### DISCUSSION TITLE
Changing Transferred In Domain Nameservers?

### DISCUSSION THREAD

**Discussion by heyjordn** [vercel/community#186] [Help] [answered]
Hi,

I recently transferred in a domain to Vercel from a different provider, however, the nameservers are still saying the third party and the original Provider says the domain is no longer on their server. How do I change the nameservers on Vercel?

Screenshot from the dashboard

<img width="413" alt="Screen Shot 2021-11-22 at 3 19 05 PM" src="https://user-images.githubusercontent.com/4820517/142929398-9e9108d4-2740-44f5-9016-e4301b2c415e.png">

How could I change the nameservers to Vercel's own if the domain has already been transferred in?

<img width="997" alt="Screen Shot 2021-11-22 at 3 23 20 PM" src="https://user-images.githubusercontent.com/4820517/142930350-9591e1f2-9ba0-473b-beb5-3c24a1924398.png">

**Comment by Pieparker** [comment, marked as answer]
Hey there! You'll want to ensure that you update the `NS` records at your registration to match the ones our UI provides to you. It's been a while since we heard from you so I'll presume that you got to the root of this issue!

If issues persist or a future reader stumbles upon this discussion and is exhibiting the same problem, please create a new discussion.
